static/docs/get-started/example-pipeline.md

```diff
@@ -38,7 +38,7 @@ your browser to download `code.zip`.
 ```dvc
 $ mkdir example && cd example
 $ git init
-$ wget https://dvc.org/s3/examples/so/code.zip
+$ wget https://dvc.org/s3/code/tutorial-nlp/code.zip
```

Comment:
where do we store and take code for the actual "big" tutorial?

Reply:
I would use code/example-nlp or something

Reply:
for now at least, but I see your point ... you are trying to unify them and make them tutorials

Reply:
we should use then `data/tutorial/nlp` and `data/tutorial/dl` (or `data/tutorial/versioning` is fine)

Reply:
`dl` stands for deep learning

Reply:
Yeah that change wasn't right. Just updated it in d3ac9dd because there are 2 code files for tutorial-nlp. I added both code files there because you said example-pipeline + tutorial will become a single NLP tutorial. (See #564.)

Reply:
> we should use then data/tutorial/nlp and data/tutorial/dl (or data/tutorial/versioning is fine). dl stands for deep learning

Hmmm so example-pipeline will be its own tutorial? I thought you said (on Slack to me directly) only example-versioning will become a standalone tutorial.

Reply:
no, no ... you are right, I think there will be one nlp tutorial, we don't need both ... it's just naming - it can be `data/tutorials/nlp` instead of `data/tutorial-nlp` - a more structured way

Reply:
OK. Updated the bucket file structure, and the code in 45c221d